utils/pofix.py

```diff
@@ -175,6 +175,79 @@
+("also be prepared to spend quite some time here — mining", "also, be prepared to spend quite some time here — mining"),
+("do not know where it was found, but Baglur said", "know not exactly where it can be found, but Baglur said"),
+("Here is some of the coal that we need! Bring the miners to take it!", "Here is some of the coal that we need! Send a miner here to take it!"),
+("Here is the mine of precious gold! Send the miners this way.", "Here is the mine of precious gold! Send a miner here to take it!"),
+("So now go to the forge and make your sceptre.", "Now go to the forge and make your sceptre."),
+("yes, very impressive ruby... Even in this raw form", "yes, very impressive ruby... even in this raw form"),
+("Maybe... maybe... It will be hard, let me see...", "Maybe... maybe... it will be hard, let me see..."),
+("no... unfortunately... It seems it can’t", "no... unfortunately... it seems it can’t"),
+("Another group of dwarves that live south of here... Above ground, if", "Another group of dwarves that live south of here... above ground, if"),
+("Getting this cut isn’t worth that much; what with", "Getting this cut isn’t worth that much — what with"),
+("all the other expenses, if we pay more than a thousand", "all the other expenses. If we pay more than a thousand,"),
+("Look at him fly... If only we could move that fast.", "Look at him fly... if only we could move that fast."),
+("I thought we left the elves behind at the gates way back when... It seems", "I thought we left the elves behind at the gates way back when... it seems"),
+("Ha! Now we can get those tools easily, and go back to our own caves.", "Ha! Now we can easily take their tools back to our own caves."),
+("True. I suspect these elves will try to besiege us here.", "True. I suspect these elves will try to besiege us."),
+("Shorbear tools. Eventually the sceptre was crafted, but ", "Shorbear tools. Eventually, the sceptre was crafted, but "),
+("not right, no matter what enchantment of runes", "not right no matter what enchantment of runes"),
+("I see... Well, we will not be fooled by your haggling", "I see... well, we will not be fooled by your haggling"),
+("I have made it past those elves, but they will chase me, and elvish", "I have made it past these elves, but they will chase me, and elvish"),
+("We will certainly help any warrior of Wesnoth who needs our help!", "We will certainly help any warrior of Wesnoth in need!"),
+("I’m a dragoon, with Haldric II’s personal bodyguard.", "I’m a dragoon with Haldric II’s personal bodyguard."),
+("I’m a cavalier, with Haldric II’s personal bodyguard.", "I’m a cavalier with Haldric II’s personal bodyguard."),
+("Thus Alanin escaped from his elvish pursuers.", "Thus, Alanin escaped from his elvish pursuers."),
+("Stop that! You leave my pets alone!", "Stop that! Leave my pets alone!"),
+("Nothing!? Even this magical piece of crap lets me down!", "Nothing!? Even this magical piece of troll dung lets me down!"),
```

Comment:
Although the basic meaning doesn't change here, I think maybe it shouldn't be in pofix… it's a flavour change…